### ISSUE TITLE
More Screenshots overlay breakage in the last releases of 60 and 61

### ISSUE THREAD

**Issue by johngruen** [mozilla-services/screenshots#4217]
So this is reported for Ghacks but i think i saw this elsewhere and it's worth doing more debugging on it.

### STR

1. Go to this article in ghacks: https://www.ghacks.net/2018/03/12/firefox-59-firefox-screenshots-gets-image-editing-functionality/
2. Scroll down the page: 
3. Try to take a screenshot

### Expected
Normal Screenshots stuff

### Actual
Clicking the iframe does some weird stuff.

![iframe break](https://user-images.githubusercontent.com/3323249/37309183-2636e324-260e-11e8-8e8c-b4d9f774f7ec.gif)

**Comment by wresuolc** [comment]
@SoftVision-CosminMuntean can you check this out?

* Does this show up on any other sites but ghacks?
* Does this happen in Firefox 58 or 59 on ghacks?

**Comment by chenba** [comment]
@SoftVision-CosminMuntean This latest patch should still also work on https://brightside.me (see #3907).  (npr.org no long exhibit the behavior originally reported on #3907.)

**Comment by SoftVision-CosminMuntean** [comment]
> Does this show up on any other sites but ghacks?

@wresuolc I have managed to reproduce this issue also on some Reddit articles ([eg: this article](https://www.reddit.com/r/iamverysmart/comments/84b7dn/message_for_all_the_people_who_think_iq_matters/)).
Probably there are more websites that are affected, but I haven't found them yet.
> Does this happen in Firefox 58 or 59 on ghacks?

I have tested this issue with ghacks and reddit sites on all Firefox channels. Here are the results:
- Firefox 58.0.2 release (Screenshots 25.0.0) - the issue is not reproducible
- Firefox 59.0 release (Screenshots 25.0.0) - the issue is not reproducible
- Beta 60.0b3 (Screenshots 30.0.0) - the issue is reproducible 
- Dev Edition 60.0b2 (Screenshots 30.0.0) - the issue is reproducible
- Nightly  61.0a1 (Screenshots 30.0.0) - the issue is reproducible

It seems that the issue is reproducible on the Firefox versions where the Screenshots version is 30.0.0. 
So Firefox 59.0 release is not affected, only from Beta 60.0b3 and up.

However, I have also tested this issue on latest Nightly (61.0a1) with latest Screenshots (31.3.0) [dev version](https://screenshots.dev.mozaws.net/homepage/install-test-local.html) and the issue is no longer reproducible on ghacks and reddit sites.


> @SoftVision-CosminMuntean This latest patch should still also work on https://brightside.me (see #3907). (npr.org no longer exhibit the behavior originally reported on #3907.)

@chenba Just tested this issue on brightside and npr sites with latest Screenshots (31.3.0) [dev version](https://screenshots.dev.mozaws.net/homepage/install-test-local.html) and the issue is not reproducible.

Please let me know if you need more information.

**Comment by SoftVision-CosminMuntean** [comment]
I have ran another round of tests using latest Screenshots 31.3.0 dev version on latest Nightly (61.0a1) build in order to verify if there are new issues or regressions.

I have verified this using the first 50 websites from https://www.alexa.com/topsites page and also a few more that we consider popular (eg: cnn.com, nytimes.com, stackoverflow.com, imdb.com, pinterest.com etc).

During testing I haven't encountered any new issues so I think we are good for uplifting this fix in Nightly. After the fix will lend in Nightly, I will test this again to make sure that I haven't missed anything and I will report back the results.
